compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -304,7 +304,7 @@ def AMDAIEPeelForLoop :
   let options = [
     Option<"peelingType", "peeling-type",
       "mlir::iree_compiler::AMDAIE::PeelingType",
-      /*default=*/"",
+      /*default=*/"mlir::iree_compiler::AMDAIE::PeelingType::FirstLast",
```

Comment:
```suggestion
      /*default=*/"::mlir::iree_compiler::AMDAIE::PeelingType::FirstLast",
```

The preceding `::` ensures that the global `mlir` namespace is used.

Reply:
If this is the case, we'd need to update the namespace all over the place in this repository. I won't address this in this PR to keep consistent. If it's really needed, I'd do it in a follow-up PR.

Reply:
Sure, it'd be great if you can address this in a follow-up PR. Not a blocker for this PR.
General code health of the repository and correct conventions go a long way in maintaining the same.